clang/include/clang/Basic/Builtins.td

```diff
@@ -662,6 +662,12 @@ def Clz : Builtin, BitShort_Int_Long_LongLongTemplate {
 
 // FIXME: Add int clzimax(uintmax_t)
```

Comment:
Should this FIXME and the `ctzimax(uintmax_t)` one below be removed?

Reply:
Probably; it's unlikely we're going to add them at this point.